backend/docs/API_Specifications/facility-v2.yaml

```
@@ -0,0 +1,382 @@
+          description: The workflow status of the facility creation. Facilities may need to be approved for creates and edits before they become active in the system. 
+        additionalDetails:
+          type: jsonb
+          description: Extra details to be added as key value attribute pairs as needed. To be used only post release by implementation teams for customization requirements. Not to be used to store UI values etc..
+        isActive:
+          type: boolean
+          description: Whether a facility is active or not.
+
+    HealthFacilityDetails:
+      description: These are health facility specific attributes. This needs to be converted into a JSON schema and added to MDMS. All facilities of type "Health" to be verified against this schema.
+      hfr_id:
+          type: string
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)